src/components/FoodBox.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+class FoodBox extends React.Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            count: 1
+        }
+    }
+    handleQuantity = (event) =>{
+        this.setState({
+            count: event.currentTarget.value
+        })
+    }
+    render(){
+        return (
+            <div className="box">
+                <article className="media">
+                    <div className="media-left">
+                    <figure className="image is-64x64">
+                        <img src={this.props.food.image} alt='img'/>
```

Comment:
when you have multiple props elements being passed, it is good practice to destructure them before the return to access the variables instead of doing props. ... everytime.

Reply:
Nicee! I'm gonna note it!